src/shapes/Group.ts

```diff
@@ -821,8 +840,8 @@ export class Group extends createCollectionMixin(FabricObject<GroupEvents>) {
         strokeWidth: 0,
       }),
       bboxSizeAfter = this._getTransformedDimensions({
-        width: bbox.width,
-        height: bbox.height,
+        width: w,
+        height: h,
```

Comment:
might be this is an unwanted change?
bbox.width/height could have been undefined if group has no objects and then `_getTransformedDimensions` would use this.width/height

Reply:
i don't even know how we eneded up pass width and height to `_getTransformedDimensions` , it accepted skewX and skeW for the only reason to cheat during skew control support, but shouldn't take anything to be honest.

Regarding your question:

```
const width = hasWidth ? this.width : w,
      height = hasHeight ? this.height : h,
```
would using width and height local vars effectively sort the issue?
If bbox does not return width/height ( but why???) it would have this.width and this.height.

Reply:
In #8767 I changed this piece of logic. And I want a bit more so we shouldn't dig into this here IMO.
The only reason it wouldn't return is if group has no children.
Really, this part is not good enough in general.
width/height from props should take affect only if the layout type is `fixed`. All other conditions should ignore it.
The center point however is affected by left/top for all cases.
What I want to do is first set the center point and after that preform the initial layout.
I think we can move forward with this even if we commit a regression since obviously we need to look into this anyways and the BBox approach should reduce the layout logic by half and enable 10 times of options.

Reply:
Ok so i ll remove this change here.